README.md

```diff
@@ -133,7 +133,7 @@ In this flow, we’re reusing static resources themselves as dictionaries that w
 ### Dictionary options header
 The `use-as-dictionary:` response header is a [structured field dictionary](https://www.rfc-editor.org/rfc/rfc8941.html#name-dictionaries) that allows for setting multiple options and for future expansion.  The supported options and defaults are:
 
-* **match** - URL-matching pattern for the dictionary to apply to. *Required*. This is parsed as a [URL](https://url.spec.whatwg.org/) that relative or absolute URLs as well as `*` wildcard expansion. e.g. `/app1/main*` will match `https://www.example.com/app1/main_12345.js` and `main*` in response to `https://www.example.com/app1/main_1.js` will match `https://www.example.com/app1/main.xyz.js`.
+* **match** - URL-matching pattern for the dictionary to apply to. *Required*. This is a patternString for a [URLPattern](https://wicg.github.io/urlpattern/) `URLPattern(patternString, baseURL)` constructor where the baseURL is the URL of the request and where support for [regexp](https://wicg.github.io/urlpattern/#token-type-regexp) tokens is disabled. URLPattern allows for absolute or relative URLs and the resulting pattern MUST be same-origin as the request. e.g. `/app1/main*` will match `https://www.example.com/app1/main_12345.js` and `main*` in response to `https://www.example.com/app1/main_1.js` will match `https://www.example.com/app1/main.xyz.js`.
```

Comment:
I think it's slightly ill-formed to say that a pattern "must be same-origin"; must it match _only_ same-origin URLs, or is any pattern that could match a same-origin URL valid? It seems like accepting URL patterns which might match cross-origin URLs is not going to break anything, and might just be something to warn developers about if it seems likely they intended to do something invalid.